styles.css

```diff
@@ -0,0 +1,588 @@
+    font-size: 16px;
+    background: #e51937;
+    color: #fff;
+  }
+
+  .main__vector {
+    margin-top: 52px;
+  }
+}
+
+@media (max-width: 585px) {
+  .body {
+    margin: 0;
+    max-width: 585px;
+  }
+
+  .main__h1 {
+    font-size: 75px;
+  }
+
+  .main__inner {
+    width: 80%;
+    margin-bottom: 38px;
+  }
+
+  .main__img-speed {
+    margin-top: 0;
+    width: 40.14px;
+    height: 29.7px;
+    margin-right: 13px;
+  }
+
+  .main__inner-road {
+    padding-left: 0;
+  }
+
+  .main__img-road_dasktop {
+    margin: 5px 9px 0 0;
+    width: 34px;
+    height: 53px;    }
+
+  .main__text-big {
+    font-size: 27px;
+    line-height: 33px;
+  }
+
+  .main__text-norm {
+    font-size: 15px;
+    line-height: 23px;
+    padding-left: 9px;
+  }
+
+  .main__text-small {
+    margin: 0;
+    font-weight: 400;
+    font-size: 10px;
+  }
+
+  .main__div-testdrive {
+    width: 70%;
+  }
+
+  .main__img-testdrive {
+    width: 19px;
+  }
+
+  .main__link {
+    font-size: 10px;
+    line-height: 20px;
+  }
+
+  .testdrive__wrapper {
+    margin: 0 auto;
+    width: 47%;
+  }
+
+  .main__button {
+    width: 44%;
+    line-height: 36px;
+    font-size: 10px;
+  }
+}
+
+@media (max-width: 375px) {
+  .body {
+    margin: 0;
+    max-width: 375px;
+  }
+
+  .header {
+    margin: 0 auto;
+    width: 100%;
+    align-items: center;
+    margin-bottom: 7px;
+  }
+
+  .header__logo-img {
+    width: 138px;
+    height: 42px;
+  }
+
+  .header__nav-link {
+    display: none;
+  }
+
+  .header__menu-burger {
+    width: 24px;
+    height: 15px;
+  }
+
+  .main__h1 {
+    font-size: 50px;
+    line-height: 57px;
+    margin-top: 20px;
+  }
+
+  .main__span {
+    margin-bottom: 45px;
+    font-size: 10px;
+  }
+
+  .main__img-background {
+    margin-top: 106px;
+  }
+
+  .main__img-tesla {
+    margin: 0;
+    width: 100%;
+  }
+
+  .main__title {
+    margin-bottom: 21px;
+  }
+
+  .main__vector {
+    margin-top: 22px;
+  }
+}
+
+@media (max-width: 330px) {
+  .body {
+    margin: 0;
+    max-width: 330px;
+  }
+
+  .main__h1 {
+    font-size: 40px;
+  }
+  .main__inner {
+    width: 80%;
+    margin-bottom: 38px;
+  }
+
+  .main__img-speed {
+    margin-top: 0;
+    width: 25px;
+    margin-right: 5px;
+  }
+
+  .main__inner-road {
+    padding-left: 0;
+  }
+
+  .main__img-road_dasktop {
+    margin: 0;
+    width: 34px;
+    height: 35px;
+  }
+
+  .main__text-big {
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .main__text-norm {
+    font-size: 10px;
+    line-height: 23px;
+    padding-left: 3px;
+  }
+
+  .main__text-small {
+    margin: 0;
+    font-size: 8px;
+  }
+
+  .main__div-testdrive {
+    width: 85%;
+  }
+
+  .main__img-testdrive {
+    width: 17px;
+  }
+
+  .main__link {
+    font-size: 10px;
+  }
+
+  .testdrive__wrapper {
+    margin: 0 auto;
+    width: 47%;
+  }
+
+  .main__button {
+    width: 47%;
+    font-size: 10px;
+  }
+    .vector {
+      padding: 6px;
+      border-width: 0 2px 2px 0;
+    }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/max_osipov/pullRequest?prid=5575857)